### PR TITLE
Added Migrations AccessLevels & Mangas

### PR DESCRIPTION
**Pull request by Kokosnootje**
Migrations AccessLevel & Manga has been added

### CODE REVIEW

**Comment by DarshanAIakhramsing** [comment]
Skiddadle Skidoodle our db is now a noodle

File: database/migrations/2018_05_25_100146_create_mangas_table.php [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+     */
+    public function up()
+    {
+        Schema::create('mangas', function (Blueprint $table) {
+            $table->increments('id');
+            $table->timestamps();
+            $table->string('title');
+            $table->int('chapters_total');
+            $table->string('discription');
+            $table->date('published')
```
Comment (okapifan) [inline database/migrations/2018_05_25_100146_create_mangas_table.php]: ;?

File: database/migrations/2018_05_25_102018_create_access_levels_table.php [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('access_levels', function (Blueprint $table) {
+            $table->increments('id');
```
Comment (okapifan) [inline database/migrations/2018_05_25_102018_create_access_levels_table.php]: Je mist de name

**Review by okapifan — APPROVED** [review]
Good Job Boy

**Comment by DarshanAIakhramsing** [comment]
I approve

**Review by DarshanAIakhramsing — APPROVED** [review]

**Pull request merged**